project.mml

```diff
@@ -48,8 +48,8 @@ Stylesheet:
 - ferry-routes.mss
 - power.mss
 - admin.mss
-- gpx.mss
-- legend.mss
+#- gpx.mss
+#- legend.mss
```

Comment:
Typo, these can be safely restored I guess.

Reply:
but they are useless for the production, no ?

Reply:
In principle, this should not change much since they have `status: off`, isn't it?

Reply:
ok